example/iterate.cpp

```diff
@@ -169,13 +167,15 @@ int main(int argc, char *argv[]) {
 	desc.add_options()
 	("group,g", boost::program_options::value<std::vector<int>>()->multitoken(), "group IDs to connect")
 	("log-file,l", boost::program_options::value<std::string>()->default_value("/dev/stderr"), "log file")
-	("log-level,L", boost::program_options::value<int>()->default_value(1), "log level")
-	("remote,r", boost::program_options::value<std::vector<std::string>>()->multitoken(), "adds a route to the given node")
+	("log-level,L", boost::program_options::value<std::string>()->default_value("info"), "log level")
+	("remote,r", boost::program_options::value<std::vector<std::string>>()->multitoken()->required(), "adds a route to the given node")
```

Comment:
As of lucid, I would drop its support, do you have any servers that old?
I think lucid doesn't even have c++11 support which is required for elliptics build